src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js

```diff
@@ -155,9 +160,12 @@ function calInboxInvitationMessageBlueBarController(
     return $q.reject(new InvalidMeetingError('Event does not involve current user.'));
   }
 
+  // This function throw an error when the event is updated
   function assertInvitationSequenceIsNotOutdated(event) {
     if (+self.meeting.sequence < +event.sequence) {
-      return $q.reject(new InvalidMeetingError('Sequence is outdated (event.sequence = ' + event.sequence + ').'));
+      self.isOutdated = true;
+
+      return $q.reject(new InvalidMeetingError('Event outdated'));
```

Comment:
You should create a local constant in this file for this message `Event outdated`, then you can use it at line 100. To avoid err-prone in the future.